pandas/tests/arrays/masked/test_arithmetic.py

```diff
@@ -165,12 +165,12 @@ def test_error_len_mismatch(data, all_arithmetic_operators):
 
 
 @pytest.mark.parametrize("op", ["__neg__", "__abs__", "__invert__"])
-@pytest.mark.parametrize(
-    "values, dtype", [([1, 2, 3], "Int64"), ([True, False, True], "boolean")]
-)
-def test_unary_op_does_not_propagate_mask(op, values, dtype):
+def test_unary_op_does_not_propagate_mask(data, op):
     # https://github.com/pandas-dev/pandas/issues/39943
-    s = pd.Series(values, dtype=dtype)
+    data, _ = data
+    if data.dtype in ["Float32", "Float64"] and op == "__invert__":
+        pytest.skip("invert is not implemented for float ea dtypes")
```

Comment:
maybe change this to add an xfail marker instead of skipping

Reply:
Set to xfail